### PR TITLE
Exhibits in exhibitions

### PR DESCRIPTION
**Pull request by jamesgorrie**
Show the exhibits within an exhibition on the exhibitions page.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev out of WIP and behind a switch.

File: common/views/components/InstallationPromo/InstallationPromo.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const InstallationPromo = ({ id, title }: Props) => (
+  <a
+    data-component='InstallationPromo'
+    data-track-event={JSON.stringify({
+      category: 'component',
+      action: 'InstallationPromo:click'
+    })}
+    id={id}
+    href={`/installations/{id}`}
+    className='plain-link promo-link bg-cream rounded-top rounded-bottom overflow-hidden flex flex--column'>
```
Comment (davidpmccormick) [inline common/views/components/InstallationPromo/InstallationPromo.js]: `rounded-corners` does the job of both `rounded-top` and `rounded-bottom` and saves you 10 keystrokes!

Comment (jamesgorrie) [reply common/views/components/InstallationPromo/InstallationPromo.js]: 👍 saved 10 keystrokes by spending keystrokes.
Ah, capitalism.

File: whats_on/app/views/components/exhibits/exhibits.njk [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+<div>
```
Comment (davidpmccormick) [inline whats_on/app/views/components/exhibits/exhibits.njk]: Could this one be a `Fragment` (assuming it's only there to stop React complaining)?

Comment (jamesgorrie) [reply whats_on/app/views/components/exhibits/exhibits.njk]: This is actually there for the async component, we're still in nunjucks and here.

**Pull request merged**